### PR TITLE
Feature/pages styling 01

### PR DESCRIPTION
**Pull request by andres-condezo**
## Feature/pages styling 01
Changes in this PR:
- Add a close button to navigation bar.
- Add styles to the close button.

### CODE REVIEW

**Review by RaoAkif — APPROVED** [review]
Hi @andres-condezo 

Your initial styling from this branch is complete! Congratulations! 🎉 But kindly wait for @Retky's review before merging :shipit: 

## To Highlight 💯 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

Congratulations-Memes-FEATURED

## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Review by Retky — APPROVED** [review]
## Hello 👋🏻 @andres-condezo
Excellent work! 💯
You have done a great job. 🥳

# Status: Approved ✔️ 



### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- You can remove all the files in the Redux folder since in other branches we recreate them.

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

---

**Pull request merged**
